javascript/javascript1/week3/travelTimeCalculate.js

```diff
@@ -0,0 +1,16 @@
+// When will we be there??
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+
+function travelTimeCalculate(travelInformation) {
+    let carSpeed = travelInformation.speed
+    let distance = travelInformation.destinationDistance
+    var hour = Math.floor(distance / carSpeed)
+    var minute = Math.ceil (60 * ((distance / carSpeed) % 1))
```

Comment:
These should be `const`.

Reply:
I'm not sure I understand your `minute` calculation.

Reply:
I am calculating 432/50 = 8.64 and 8.64 % 1 = 0.64 . End of this I am calculating how many minutes 60 * 0.64 = 38.4 and with ceil it is 39 minutes. Probably it has a simple solition too.

Reply:
Ah, ok!
Yeah you can do:
```js
const hours = Math.floor(distance / carSpeed);
const minutes = Math.ceil((60 * hours) % 60)); // which is pretty much the same, but it's a bit more clear for the reader
```

Reply:
Exactly, That is more easy to read. Thanks